.circleci/config.yml

```diff
@@ -14,4 +14,4 @@ jobs:
       - node/with-cache:
           steps:
             - run: npm install
-      - run: npm run test
\ No newline at end of file
+      - run: npm run test
```

Comment:
This is an artifact of a merge conflict - Still getting the hang of keeping everything up to date.